rid/v1/observation.yaml

```diff
@@ -122,12 +126,33 @@ components:
         timestamp:
           description: Time at which the message containing the current state information was generated by the display provider.
           type: string
+        timestamp_accuracy:
```

Comment:
This is also defined in `injection.yaml`: to factor away in `commons.yaml`. Beware however to the default, it looks like in `injection.yaml` the field is taken from v19, while you have put here the field from v22a, there is a difference in the default value.